### DISCUSSION TITLE
installation issue

### DISCUSSION THREAD

**Discussion by ghost** [metrumresearchgroup/mrgsolve#794] [Q&A]
I have  restored mac to factory setting updated to macOS Big Sur v 11.2.1, installed R version 4.0.4 (2021-02-15),  R studio Version 1.4.1103 and Xcode Version 12.4 (12D4e).

I have an issue installing mrgsolve package using both options install.packages("mrgsolve") & remotes::install_github("metrumresearchgroup/mrgsolve@dev") but was not success. 

I encountered the following issues:

1. install.packages("mrgsolve")

Installing mrgsolve [0.10.4] ...
	OK (linked cache)

2. Downloading GitHub repo metrumresearchgroup/mrgsolve@dev

Error in utils::download.file(url, path, method = method, quiet = quiet,  : 
  cannot open URL 'https://api.github.com/repos/metrumresearchgroup/mrgsolve/tarball/dev'

> library(mrgsolve)
Error: package or namespace load failed for ‘mrgsolve’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/Users/mmm/Library/Application Support/renv/cache/v5/R-4.0/x86_64-apple-darwin17.0/RcppArmadillo/0.9.900.2.0/8845285b6151967fb0d95678b7eee11a/RcppArmadillo/libs/RcppArmadillo.so':
  dlopen(/Users/mmm/Library/Application Support/renv/cache/v5/R-4.0/x86_64-apple-darwin17.0/RcppArmadillo/0.9.900.2.0/8845285b6151967fb0d95678b7eee11a/RcppArmadillo/libs/RcppArmadillo.so, 6): Library not loaded: /usr/local/gfortran/lib/libgfortran.5.dylib
  Referenced from: /Users/mmm/Library/Application Support/renv/cache/v5/R-4.0/x86_64-apple-darwin17.0/RcppArmadillo/0.9.900.2.0/8845285b6151967fb0d95678b7eee11a/RcppArmadillo/libs/RcppArmadillo.so
  Reason: image not found

Thank you

**Comment by kylebaron** [comment]
@melagha I am moving this to our discussion area: https://github.com/metrumresearchgroup/mrgsolve/discussions

**Comment by kylebaron** [comment]
> Library not loaded: /usr/local/gfortran/lib/libgfortran.5.dylib

Did you install the fortran compiler?
https://github.com/fxcoudert/gfortran-for-macOS/releases
Get the 8.2: 
https://github.com/fxcoudert/gfortran-for-macOS/releases/tag/8.2

**Reply by ghost** [reply]
It worked after installing fortran compiler recommended .
Best regards
